data-raw/countries.R

```diff
@@ -0,0 +1,29 @@
+# Get a list of countries and codes from ONS API
+
```

Comment:
Maybe add a year parameter at the start of this script and then derive the rest of the uses of year from that?

Reply:
Done